pkg/clients/dynatrace/settings_enrichment.go

```diff
@@ -29,20 +29,14 @@ type RulesResponseValue struct {
 	Rules []dynakube.EnrichmentRule `json:"rules"`
 }
 
-func (dtc *dynatraceClient) GetRulesSettings(ctx context.Context, kubeSystemUUID string) (GetRulesSettingsResponse, error) {
+func (dtc *dynatraceClient) GetRulesSettings(ctx context.Context, kubeSystemUUID string, entityID string) (GetRulesSettingsResponse, error) {
 	if kubeSystemUUID == "" {
 		return GetRulesSettingsResponse{}, errors.New("no kube-system namespace UUID given")
 	}
 
-	monitoredEntities, err := dtc.GetMonitoredEntitiesForKubeSystemUUID(ctx, kubeSystemUUID)
-	if err != nil {
-		log.Info("failed to retrieve MEs")
-
-		return GetRulesSettingsResponse{}, err
-	}
-
-	if len(monitoredEntities) == 0 {
-		log.Info("no MEs found, skipping getting enrichment rules")
+	if entityID == "" {
+		// if monitored entities were empty this field also stays empty, we return with no error
+		log.Info("No Monitored Entity ID, skip loading/getting/querying enrichment rules")
```

Comment:
that was meant as suggestion for you to choose one verb, not put all of them 😄

Reply:
fixed in https://github.com/Dynatrace/dynatrace-operator/pull/3447/commits/7806dfbbe1fd6bfb933a74ecc937edf987f1c81b